mhs-reference-implementation/selenium_tests/page_objects/build_scr.py

```diff
@@ -0,0 +1,120 @@
+import datetime
```

Comment:
So this file is the same as the file in supplier_example. I'm not sure if that's an issue or not, in my head I feel like it isn't as that's just some example POC thingy and this is the actual deliverable thing (hopefully).